### PR TITLE
Personal page

### PR DESCRIPTION
**Pull request by pandasumm**
Pull request not ready for user testing, wait to review.

### CODE REVIEW

**Comment by avaid96** [comment]
Hey Xiao, just wondering. Did you see this? Any reason why you chose not to use it?
https://ionicframework.com/docs/storage/

Seems like a much better cross platform solution. i tried it as a sample and works really well for me on web+android

**Comment by pandasumm** [comment]
I have to say I didn't see this. That's will be cool if this could work in
both web and Android. Let's move to this solution.

**Comment by avaid96** [comment]
merged to resolve conflicts

File: package.json [hunk 1]
```diff
@@ -22,12 +22,22 @@
...
     "@angular/platform-browser-dynamic": "5.2.9",
     "@ionic-native/core": "4.6.0",
+    "@ionic-native/file": "^4.7.0",
     "@ionic-native/splash-screen": "4.6.0",
     "@ionic-native/status-bar": "4.6.0",
     "@ionic/storage": "2.1.3",
     "@types/google-maps": "^3.2.0",
     "angularfire": "^2.3.0",
     "angularfire2": "^5.0.0-rc.6.0",
+    "cordova-android": "7.0.0",
```
Comment (avaid96) [inline package.json]: do we really need these? can we get rid of unused modules?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -20,6 +20,9 @@ import { AngularFireDatabaseModule } from 'angularfire2/database';
 import { AngularFireModule } from 'angularfire2';
 import { FirebaseProvider } from './../providers/firebase/firebase';
 // import { FirebaseProvider } from '../providers/firebase/firebase';
+// for storage
+import { File } from '@ionic-native/file';
```
Comment (avaid96) [inline src/app/app.module.ts]: Do we use File at all? Can we get rid of it this import?

File: src/app/app.module.ts [hunk 3]
```diff
@@ -59,6 +63,7 @@ const firebaseConfig = {
     GoogleMapPage
   ],
   providers: [
+    File,
```
Comment (avaid96) [inline src/app/app.module.ts]: Same here

File: src/pages/google-map/google-map.ts [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
 
 /**
  * Generated class for the GoogleMapPage page.
  *
  * See https://ionicframework.com/docs/components/#navigation for more info on
  * Ionic pages and navigation.
  */
-
+declare var google; //add it to work in my laptop, feel free to remove it. Xiao
```
Comment (avaid96) [inline src/pages/google-map/google-map.ts]: We'll figure this out when we meet

File: src/pages/stop-details/stop-details.ts [hunk 5]
```diff
@@ -27,10 +28,12 @@ export class StopDetailsPage {
   fav: boolean;
   stopLocation : {lat: number, long: number};
 
+  // <<<<<<< HEAD
```
Comment (avaid96) [inline src/pages/stop-details/stop-details.ts]: remove this line please

File: src/pages/stop-details/stop-details.ts [hunk 6]
```diff
@@ -141,15 +142,31 @@ export class StopDetailsPage {
...
+    };
+
+
+    let mapModal = this.modalCtrl.create(GoogleMapPage, stopInfo);
     mapModal.present();
   }
 
-  buttonClick() {
+  addFav(stopName) {
+    var tempData = ',' + stopName;
```
Comment (avaid96) [inline src/pages/stop-details/stop-details.ts]: not a huge fan of this data structure, **but it's fine for now.** 

ideally should have used something like this if we could store an object as the value of a key
'favstops': {
'weber arch': true,
'tech institute': true,
...
}

File: src/pages/stop-details/stop-details.html [hunk 7]
```diff
@@ -20,20 +20,30 @@
       <ion-title align="center">{{this.stopName}}</ion-title>
 
       <ion-buttons end>
+        <!-- <<<<<<< HEAD -->
 
-      <button ion-button clear icon-only class="googleMapPin" (click)="MapbuttonClick()">
+        <button ion-button clear icon-only class="googleMapPin" (click)="MapbuttonClick()">
 
+<!-- ======= -->
```
Comment (avaid96) [inline src/pages/stop-details/stop-details.html]: why are these comments left in? I think we should remove all of the comments in this file

File: src/pages/google-map/google-map.ts (hunk 4 above)
Comment (avaid96) [reply src/pages/google-map/google-map.ts]: It doesn't break anything on my laptop so shouldn't break on others also, I guess we can just keep it in

**Comment by avaid96** [comment]
Merging this

**Pull request merged**